dataflow/src/main/java/org/checkerframework/dataflow/cfg/block/RegularBlockImpl.java

```diff
@@ -39,6 +39,11 @@ public void addNodes(List<? extends Node> ts) {
         return Collections.unmodifiableList(contents);
     }
 
+    @Override
+    public List<Node> getNodes() {
```

Comment:
Instead of now having two methods, `getNodes` and `getContents`, in regular blocks, I would pick one name. I've gotten used to `getContents`, but `getNodes` is also fine with me.
So if you prefer `getNodes`, I would deprecate `getContents` for removal to not have this duplication.

Reply:
Good suggestion.  I did that, and that led me to notice other cleanups that could be performed (eg, a client that implemented getNodes and can now use the new method).  I also added a `getLastNode()` method since multiple clients implemented it.

I chose "getNodes" because I think it's more descriptive to clients.  (I see why getContents was chosen -- because a private field is named `contents`.)

Do you want to review the latest changes, or should I merge this?